projects/cardiovascular-patient-app/src/app/report-bp/report-bp.component.html

```diff
@@ -0,0 +1,51 @@
+<mat-card>
+  <mat-card-content>
+    <form [formGroup]="form" class="bpform" id="entryForm" (ngSubmit)="onSubmit()">
+      <section class="bpform-section">
+        <div class="systolic-input">
+          <mat-form-field class="input-width">
+            <mat-label> Systolic : </mat-label>
```

Comment:
A minor issue, but please remove the ` : ` from form field labels. Material Design does not use this and it confuses the meaning of the `*` that is displayed for required fields. I know you were probably trying to match the input fields in other components, but we should go back and fix those other ones too (in a different PR).

Reply:
Okay, I will remove : from field labels.